src/main.cpp

```diff
@@ -215,6 +219,17 @@ int main(int argc, char *argv[])
     {
         channel->sendMessage("CLEAR_FILES");
     }
+    if (!unloadFiles.empty())
+    {
+        channel->sendMessage(QByteArray("UNLOAD_FILES\n") + unloadFiles.c_str());
+    }
+    if (posX != -DBL_MAX)
+    {
+        channel->sendMessage("SET_VIEW_POSITION\n" +
+                             QByteArray().setNum(posX, 'g', 32) + "\n" +
+                             QByteArray().setNum(posY, 'g', 32) + "\n" +
+                             QByteArray().setNum(posZ, 'g', 32));
```

Comment:
I think these would make most sense as `setNum(posX, 'e', 17)` which is guaranteed to reproduce the incoming floating point number exactly, barring special values like NaN, Inf.